src/Health/HealthController.ts

```diff
@@ -98,6 +109,12 @@ export class HealthController {
     return pickWalletInfoKeys(await this.bitcoinCore.getWalletInfo())
   }
 
+  private async addBalanceLowBool(walletInfo: WalletInfo): Promise<WalletInfo> {
```

Comment:
Looks like you got part of the logic unit tested, I think we have more testable logic though. I think we can test it all including the destructuring and adding a new property to a new object if we make a function that takes a wallet and a lowWalletAmount.

```ts
type addWalletIsBalanceLow = (lowBalanceAmount: number) => (wallet: WalletInfo) => WalletInfo
```

We can call that function from our Promise function that knows about the configuration.

Reply:
Ok cool. I updated this.